crates/cli_utils/Cargo.toml

```diff
@@ -14,6 +14,7 @@ roc_collections = { path = "../compiler/collections" }
 roc_reporting = { path = "../reporting" }
 roc_load = { path = "../compiler/load" }
 roc_module = { path = "../compiler/module" }
+roc_utils = { path = "../utils" }
```

Comment:
The centralized commands `cargo()`, `zig()` and `clang()` were put in `roc_utils`.